api-misc/src/main/java/org/zalando/nakadi/config/RepositoriesConfig.java

```diff
@@ -25,7 +25,7 @@ public class RepositoriesConfig {
 
     private static final Logger LOG = LoggerFactory.getLogger(RepositoriesConfig.class);
 
-    @Profile({"acceptanceTest", "local"})
+    @Profile({"acceptanceTest", "local", "sidecar"})
```

Comment:
`sidecar` is the mode of execution, but it is very related to the deployment setup. could you achieve the same without adding another profile?

Reply:
We could work on the naming here, but I'd like to add a new profile as `acceptanceTest` and `local` can be misleading and future changes can break Archiver